scripts/py_matter_idl/matter_idl/data_model_xml/handlers/parsing.py

```diff
@@ -104,8 +104,14 @@ def NormalizeDataType(t: str) -> str:
 
 # Handle odd casing and naming
 _CASE_RENAMES_MAPPING = {
+    "amperage_mA": "amperage_ma",
```

Comment:
So are we not using the spec casing for these types here because of ZAP limitations?  Is there something tracking that?